plugins/modules/lambda_event.py

```diff
@@ -78,6 +78,13 @@
           - Required when I(event_source=stream).
         choices: [TRIM_HORIZON,LATEST]
         type: str
+      function_response_types:
+        description:
+          - (Streams and Amazon SQS) A list of current response type enums applied to the event source mapping.
+        type: list
+        elements: str
+        choices: [ReportBatchItemFailures]
```

Comment:
only one choice and using a list of strings? 
Could you please update the parameter and define it as a boolean?

Reply:
This is coming from the AWS spec. Presumably, they are doing this with the intent that there will be more options in the future. I think we should leave this as is, because it will be much easier to add a new item to this list than to change the parameter type altogether in the future.

Reply:
@abikouo , yes assuming AWS will come up with more choices in future, I kept it as per [boto3 documentation](https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/lambda.html#Lambda.Client.create_event_source_mapping).